src/utils.js

```diff
@@ -29,6 +29,34 @@ function qualifiedNameToId(name) {
     return name.replace(/\//g, '~');
 }
 
+/**
```

Comment:
Great docs! 

My only comment is maybe we should call it `qualifiedNameFromParameterId()`. For reasons I don't understand Yamcs provides us with a `qualifiedName` from the mdb, but the parameters come back with an `id` split into `namespace` and `name`. WHO KNOWS.